Assets/Scripts/DoorBehaviour.cs

```diff
@@ -23,10 +23,12 @@ public class DoorBehaviour : MonoBehaviour
     public string DoorSqueak;
     [FMODUnity.EventRef]
     public string DoorSlam;
+    public FMODUnity.StudioEventEmitter TutorialMusic;
```

Comment:
Might be better to play this in the tutorial logic itself instead. I'll msg you on discord where